<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-25T12:09:44 GMT-07:00" name="Suite" finished-at="2024-09-25T12:10:09 GMT-07:00" duration-ms="25844">
    <groups>
    </groups>
    <test started-at="2024-09-25T12:09:44 GMT-07:00" name="Login Functionality" finished-at="2024-09-25T12:10:09 GMT-07:00" duration-ms="25844">
      <class name="Tests.TestLogin">
        <test-method is-config="true" signature="beforeLogin()[pri:0, instance:Tests.TestLogin@3e58a80e]" started-at="2024-09-25T12:09:44 GMT-07:00" name="beforeLogin" finished-at="2024-09-25T12:09:47 GMT-07:00" duration-ms="2599" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeLogin -->
        <test-method signature="Login(java.lang.String,java.lang.String)[pri:0, instance:Tests.TestLogin@3e58a80e]" started-at="2024-09-25T12:09:47 GMT-07:00" name="Login" data-provider="getCredentials" finished-at="2024-09-25T12:10:01 GMT-07:00" duration-ms="14578" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[hfgff]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[gfhfg]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [PASS] but found [FAIL]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [PASS] but found [FAIL]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Tests.TestLogin.Login(TestLogin.java:64)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method is-config="true" signature="AfterLogin()[pri:0, instance:Tests.TestLogin@3e58a80e]" started-at="2024-09-25T12:10:02 GMT-07:00" name="AfterLogin" finished-at="2024-09-25T12:10:02 GMT-07:00" duration-ms="142" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterLogin -->
        <test-method is-config="true" signature="beforeLogin()[pri:0, instance:Tests.TestLogin@3e58a80e]" started-at="2024-09-25T12:10:02 GMT-07:00" name="beforeLogin" finished-at="2024-09-25T12:10:03 GMT-07:00" duration-ms="1287" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeLogin -->
        <test-method signature="Login(java.lang.String,java.lang.String)[pri:0, instance:Tests.TestLogin@3e58a80e]" started-at="2024-09-25T12:10:03 GMT-07:00" name="Login" data-provider="getCredentials" finished-at="2024-09-25T12:10:09 GMT-07:00" duration-ms="5890" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method is-config="true" signature="AfterLogin()[pri:0, instance:Tests.TestLogin@3e58a80e]" started-at="2024-09-25T12:10:09 GMT-07:00" name="AfterLogin" finished-at="2024-09-25T12:10:09 GMT-07:00" duration-ms="112" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterLogin -->
      </class> <!-- Tests.TestLogin -->
    </test> <!-- Login Functionality -->
  </suite> <!-- Suite -->
</testng-results>
